smurfs/src/components/Form.js

```diff
@@ -0,0 +1,49 @@
+import React, {useState} from "react";
+import {connect} from "react-redux";
+import {pushData} from "../actions"
+
+
+const Form = (props) =>{
+    const [state, setState] = useState({
+        newName: "",
+        newAge: "",
+        newHeight: "",
+        newId: ""
+    })
+
+    let handleChanges = e => {
+        setState({ 
+            newName: e.target.value,
+            newAge: e.target.value,
+            newHeight: e.target.value,
+            newId: Date.now()
```

Comment:
nice taking the extra step, but its unnecessary because of how its setup on the backend